### PR TITLE
urls: add language prefix to urls

### PR DESCRIPTION
**Pull request by Rineee**

### CODE REVIEW

File: digitalstrategie/urls.py [hunk 1]
```diff
@@ -17,23 +18,27 @@
     path('admin/', include(wagtailadmin_urls)),
     path('documents/', include(wagtaildocs_urls)),
     path('i18n/', include('django.conf.urls.i18n')),
-    re_path(r'^sitemap\.xml$', wagtail_sitemap),
     re_path(r'^robots\.txt$', TemplateView.as_view(
         template_name='robots.txt',
         content_type="text/plain"), name="robots_file"),
-    path('search/', SearchResultsView.as_view(), name="search"),
     path('newsletter-signup/', newsletter_view, name="newsletter-signup"),
```
Comment (Rineee) [inline digitalstrategie/urls.py:24]: I somehow could not make it work for the newsletter signup, but also could not test properly.. do you think this is important to also have the language there, so I should try again @fuzzylogic2000?

File: digitalstrategie/urls.py [hunk 2]
```diff
@@ -17,23 +18,27 @@
...
 
     # For anything not caught by a more specific rule above, hand over to
     # Wagtail's page serving mechanism. This should be the last pattern in
     # the list:
-    path('', include(wagtail_urls)),
+    # path('', include(wagtail_urls)),
 
     # Alternatively, if you want Wagtail pages to be served from a subpath
     # of your site, rather than the site root:
     #    url(r'^pages/', include(wagtail_urls)),
```
Comment (fuzzylogic2000) [inline digitalstrategie/urls.py:33]: Can we either move or entirely remove this?

File: digitalstrategie/urls.py (hunk 1 above)
Comment (fuzzylogic2000) [reply digitalstrategie/urls.py:24]: Hmm, we are not giving the language to the newsletter provider, right? That was in kosmo...
So, I don't think, we need the language here, but maybe we should talk to @maxliqd again.

**Review by fuzzylogic2000 — CHANGES_REQUESTED** [review]
Yes, it does add the language path, but the language switch does not work anymore. And it looks off, for some reason, the list has the dots now.
Screenshot from 2022-11-04 09-54-15

**Comment by Rineee** [comment]
> Yes, it does add the language path, but the language switch does not work anymore. And it looks off, for some reason, the list has the dots now. Screenshot from 2022-11-04 09-54-15

Yeah, the dots were there before, but looks good on dev now, so will rebase..

And the switching worked for me? Will check again!

File: digitalstrategie/urls.py (hunk 2 above)
Comment (Rineee) [reply digitalstrategie/urls.py:33]: I guess? I wasnt sure where this is coming from, so just commented out.. so should I remove it?

**Review by fuzzylogic2000 — APPROVED** [review]
Works in firefox, but not in chromium? let's try on dev!

**Pull request merged**